mods/d2k/rules/world.yaml

```diff
@@ -83,6 +83,10 @@ World:
 		Type: RockCrater
 		Sequence: rockcraters
 		SmokePercentage: 0
+	SmudgeLayer@Concrete:
+		Type: ConcreteCrater
+		Sequence: rockcraters
```

Comment:
This uses the same artwork as the rock craters, so why does it need a new layer?